### PR TITLE
Add instructions for adding to PATH

### PR DESCRIPTION
**Pull request by platonicsocrates**
Add instructions for adding to PATH + fix typo :)

Missing out the PATH is a very common error we have seen blocking our students from getting set up with the quick start. Once spotted, it's a super simple fix but easy to overlook!

### CODE REVIEW

File: content/md/en/docs/main-docs/install/troubleshooting.md [hunk 1]
```diff
@@ -122,3 +122,12 @@ rustup uninstall nightly
...
 ```
+
+### Ensure PATH is set correctly
+
+If after installing Rust the commands don't seem to work, showing errors such as `command not found: rustup`, make sure it your PATH is configured correctly.
+
+Currently, the `rustup` installer installs by default to the bash profile (on mac). If you are using another shell, make sure to add this line to your profile (e.g. `.zshrc`):
+```
+. "$HOME/.cargo/env"
+```
```
Comment (NukeManDan) [inline content/md/en/docs/main-docs/install/troubleshooting.md:133]:
```suggestion
```bash
source "$HOME/.cargo/env"
```
```

**Review by NukeManDan — CHANGES_REQUESTED** [review]
I think this is a good addition, do you agree with the change here?

File: content/md/en/docs/main-docs/install/troubleshooting.md (hunk 1 above)
Comment (platonicsocrates) [reply content/md/en/docs/main-docs/install/troubleshooting.md:133]: Awesome! Great suggestion - agreed!

**Review by lisa-parity — APPROVED** [review]
LGTM

**Pull request merged**